rust/noosphere-ns/src/records.rs

```diff
@@ -12,7 +12,7 @@ use std::{convert::TryFrom, str, str::FromStr};
 use ucan::{builder::UcanBuilder, crypto::KeyMaterial};
 use ucan::{chain::ProofChain, crypto::did::DidParser, Ucan};
 
-/// An [NSRecord] is the internal representation of a mapping from a
+/// An [NsRecord] is the internal representation of a mapping from a
```

Comment:
I'm not ready to talk about how `NsRecord`/`NSRecord` is the only struct prefixed by `NS`  :upside_down_face:

Reply:
:sweat_smile: